docs/_docs/Instructors/03-Using-nbgrader-for-Exercise-Notebooks.md

```diff
@@ -1,99 +1,83 @@
 
-### Expected output (optional) 
-While not strictly required for the Exercises to work, it is useful for the students to have some examples of what the output should be, to help them debug. These can be added as normal cells, and don't actually count for evaluation. You can simply grab some output your expect, and paste it in markdown in a cell.  This allows students to check their logic, without having to submit the notebook or reverse engineer the actual tests (especially as sometimes you might want to hide the tests to prevent sneakiness). 
+### Expected output
+This is useful in the Python prep course where students are beginning with Python. In the Academy, we assume a working level of Python and better debugging skills, so it is not necessary.
+  
+The epected output can be added as normal cells. You can take some output your expect, and paste it in markdown in a cell. This allows students to check their logic, without having to submit the notebook or reverse engineer the actual tests. See an example of a test cell and an expected output cell:
 
 <img src="https://image.ibb.co/hDdHzx/Screen_Shot_2018_04_08_at_9_11_41_PM.png" width="900" />  
 
-
 ### Autograded Test cells 
 Great, now to evaluate that the student's implementation actually works. 
 <img src="https://image.ibb.co/dbGLKx/Screen_Shot_2018_04_08_at_9_17_45_PM.png" width="900" />  
 
 A few things to notice here:   
-1. The cell is now `Autograder tests` (this is relatively easy to get wrong and write `Autograder answer`, which will give you painful debugging problems).   
-2. The cell has points. If it raises errors, the student does not get those points. Make your entire Exercise Notebook's points sum 20, so that the grading system is coherent.   
-3. In this case, some tests were hidden, and others weren't. There is generally no particular need to hide the tests, but sometimes the answer might be a string, and then it might be too tempting to look it up. Best use common sense here.  
-4. Using `math.isclose` may be better than using `isequal` or `==`. For instance, in that last example, the correct answer was `-15.999999999999996`, so we compared it to `-16`, with a tolerance of `abs_tol=0.001`
-5. If you want to hide some of the tests (because they show the answer for example), you need to write them between `### BEGIN HIDDEN TESTS` and `### END HIDDEN TESTS`.
+1. The cell is now `Autograder tests`.
+1. The cell has points. If it raises errors, the student does not get those points. Make your entire Exercise Notebook's points sum 20, so that the grading system is coherent.   
+1. In this case, some tests were hidden, and others weren't. Do not use hidden tests. If you need to hide a simple answer, use hashes.
+1. Using `math.isclose` may be better than using `isequal` or `==`. For instance, in that last example, the correct answer was `-15.999999999999996`, so we compared it to `-16`, with a tolerance of `abs_tol=0.001`
```

Comment:
```suggestion
1. The cell is now `Autograder tests`.
2. The cell has points. If it raises errors, the student does not get those points. Make your entire Exercise Notebook's points sum 20, so that the grading system is coherent.   
3. In this case, some tests were hidden, and others weren't. Do not use hidden tests. If you need to hide a simple answer, use hashes.
4. Using `math.isclose` may be better than using `isequal` or `==`. For instance, in that last example, the correct answer was `-15.999999999999996`, so we compared it to `-16`, with a tolerance of `abs_tol=0.001`
```